client/a8c-for-agencies/sections/sites/sites-dataviews/interfaces.ts

```diff
@@ -18,7 +18,3 @@ export interface SitesDataViewsProps {
 	dataViewsState: DataViewsState;
 	onRefetchSite?: () => Promise< unknown >;
 }
-
-export interface SiteInfo extends SiteData {
-	id: number;
-}
```

Comment:
My understanding of the code base is that this type was just hack to make the data views work. But in my testing it's not necessary. So let me know if I'm missing anything.